ports/stm32/boards/STM32H747I_DISCO/README.md

```diff
@@ -0,0 +1,102 @@
+# STM32H747I-DISCO Board
```

Comment:
This README is too much, none of the other boards have it.

Any critical info (eg the Ethernet hardware mod) could be kept, but would be best in `board.md` so it appears on the download page.